core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -28,14 +28,7 @@
 import hudson.Functions;
 import hudson.Util;
 import hudson.console.ConsoleLogFilter;
-import hudson.model.Computer;
-import hudson.model.Executor;
-import hudson.model.ExecutorListener;
-import hudson.model.Node;
-import hudson.model.Queue;
-import hudson.model.Slave;
-import hudson.model.TaskListener;
-import hudson.model.User;
+import hudson.model.*;
```

Comment:
Please do not squash imports in the production code. It may cause ambiguity in some edge cases.